app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -4,36 +4,74 @@
 # Make sure to set KUBELET_VERSION to an image tag published here:
 # https://quay.io/repository/coreos/hyperkube?tab=tags Alternatively,
 # override $KUBELET_ACI to a custom location.
+#
 
 set -e
 
-if [ -z "${KUBELET_VERSION}" ]; then
-    echo "ERROR: must set KUBELET_VERSION"
-    exit 1
+function require_ev_all() {
+	for rev in $@ ; do
+		if [[ -z ${!rev} ]]; then
+			echo ${rev} is not set
+			exit 1
+		fi
+	done
+}
+
+function require_ev_one() {
+	for rev in $@ ; do
+		if [[ ! -z ${!rev} ]]; then
+			return
+		fi
+	done
+	echo One of $@ must be set
+	exit 1
+}
+
+if [[ ! -z ${KUBELET_VERSION} ]]; then
+	echo KUBELET_VERSION environment variable is deprecated, please use KUBELET_IMG_TAG instead
+fi
+
+if [[ ! -z ${KUBELET_ACI} ]]; then
+	echo KUBELET_ACI environment variable is deprecated, please use the KUBELET_IMG instead
```

Comment:
I think it should be "please use the KUBELET_IMG_URL instead" -- that var is closer to KUBELET_ACI use.